### DISCUSSION TITLE
> CefSharp ver 65 support drag and drop with out having to write any separate function for it. But it is not supporting Google Map API anymore for which I have to migrate to ver 98 or above. But with ver 98 drag and drop does not work.

### DISCUSSION THREAD

**Discussion by SUHASABOROLE** [cefsharp/CefSharp#4810] [General]
> CefSharp ver 65 support drag and drop with out having to write any separate function for it. But it is not supporting Google Map API anymore for which I have to migrate to ver 98 or above. But with ver 98 drag and drop does not work.
> 
> Give me an example where I can use CefSharp ver98 and above and still drag and drop images or youtube videos from the cefsharp browser to my InkCanvas

_Originally posted by @SUHASABOROLE in https://github.com/cefsharp/CefSharp/issues/4592#issuecomment-2128653039_

**Comment by amaitland** [comment]
Probably a duplicate of #4801

**Comment by SUHASABOROLE** [comment]
Any Update on  drag and drop on browser

**Comment by amaitland** [comment]
https://github.com/cefsharp/CefSharp/issues/4592#issuecomment-1806302269
